_includes/site-card.html

```diff
@@ -0,0 +1,11 @@
+<div class="usa-card__container">
```

Comment:
**polish**:  There's no set convention (_that I've found_) for comments. 

Either way, it'd be a nice enhancement to add some component documentation. Improvements to formatting are welcome.

```suggestion
{% comment %}
  Single card element for `site-card-list`.
  
  Variables:
    - heading: String heading for a single card.
    - body: String body that gets placed in a paragraph tag.
    - linkUrl: String for the link url.
    - linkText: String link label.
{% endcomment %}
<div class="usa-card__container">
```